<launch>
	<arg name="sub_topic"/>
        <arg name="pub_topic"/>
	<arg name="my_mergemap_frame"/>
	<arg name="other_robot_name"/>

	<node pkg="roscore_communication" type="get_other_pose" name="get_$(arg other_robot_name)_pose" output="screen">
		<param name="sub_topic" value="$(arg sub_topic)"/>
		<param name="pub_topic" value="$(arg pub_topic)"/>
		<param name="my_mergemap_frame" value="$(arg my_mergemap_frame)"/>
	</node>
</launch>
